vision.md

```diff
@@ -103,16 +125,17 @@ C#, embedded systems (IT focused)
       up-to-date (while important to him not always #1 prio to overall
       initiative/project)
 
-- Special case : embedded dev with OT focus
-    - familiar with PLCs, SCADA systems,
-    - dealing with emerging need for connectivtiy and IIoT
-    - key concern is security, robusteness and resource efficiency which is
-      usually overruling any "typical" IT solution on the device and implies
-      some kind of custom logic. (e.g. rather closed OS, no dependencies can be
-      installed, very strict certification and QA process , no CI/CD possible,
-      long prodcut lifecycle 10-20 years)
+An "Embedded Developer" with OT focus is an example of a "Plugin Developer":
+
+- familiar with PLCs, SCADA systems,
+- dealing with emerging need for connectivity and IIoT
+- key concern is security, robusteness and resource efficiency which is
+  usually overruling any "typical" IT solution on the device and implies
+  some kind of custom logic. (e.g. rather closed OS, no dependencies can be
+  installed, very strict certification and QA process, no CI/CD possible,
+  long prodcut lifecycle 10-20 years)
 
-**Summary:** The persona types adressed by thin-edge.io often have conflicting
+The persona types adressed by thin-edge.io often have conflicting
```

Comment:
Personal note. I really don't like the term "persona" when used in a document addressed to a large community of readers. This a UX engineers specific term which adds nothing more than the word "user" when used as here only to distinguish roles. 

```suggestion
The users addressed by thin-edge.io often have conflicting
```